pkg/controllers/multicluster/remotecluster_controller.go

```diff
@@ -202,12 +205,26 @@ func (r *RemoteClusterReconciler) constructClusterManagerRuntime(remoteCluster *
 		return nil, err
 	}
 
+	subnetSet := sets.NewCallbackSet()
+
+	var subnetList *networkingv1.SubnetList
+	if subnetList, err = utils.ListSubnets(managerRuntime.GetAPIReader()); err != nil {
```

Comment:
Maybe simply list all the subnets in remote cluster will get some conflict subnets?

Reply:
And clusters with conflict subnets won't be ready.